src/controller/AdminController.php

```diff
@@ -45,11 +45,12 @@ public function displayAllElements(UserModel $userModel, PostModel $postModel, C
 		// display 
 		$this->render(
```

Comment:
Je viens de voir qu'en faite ton $_SESSION est presque toujours présent dans ton appel de fonction $this->render. Peut être vaut il mieux ne pas le mettre otionnel et le mettre en deuxième position de tes paramètres.

Reply:
$_SESSION mis en deuxième position dans la fonction et n'est plus optionnel